ioctl/newcmd/alias/aliasremove_test.go

```diff
@@ -13,37 +16,39 @@ import (
 )
 
 func TestNewAliasRemoveCmd(t *testing.T) {
+	require := require.New(t)
 	// mock a client
 	ctrl := gomock.NewController(t)
 	client := mock_ioctlclient.NewMockClient(ctrl)
-	client.EXPECT().SelectTranslation(gomock.Any()).Return("%s is removed", config.English).Times(6)
-
-	// configuration files to temporary files
-	defaultConfigFile := config.DefaultConfigFile
-	config.DefaultConfigFile = filepath.Join(os.TempDir(), "config.default")
-	defer func() {
-		_ = os.Remove(config.DefaultConfigFile)
-		config.DefaultConfigFile = defaultConfigFile
-	}()
-
 	cfg := config.Config{
```

Comment:
add Wallet = tempDir . because this command use client.DeleteAlias to write default config file.